Sources/TensorFlow/Initializers.swift

```diff
@@ -453,6 +453,31 @@ public extension Tensor where Scalar: TensorFlowFloatingPoint {
     }
 }
 
+public extension Tensor where Scalar: TensorFlowIndex {
+    /// Creates a tensor by drawing samples from a categorical distribution.
+    ///
+    /// - Parameters:
```

Comment:
This should use the same documentation comment formatting as the other doc comments in this repo. In particular:
* Put all the parameters under "Parameters". Swift doesn't have a concept of "Attrs".
* Name the return section "Returns" instead of "Output".
* Rename the `batch_size`, `num_classes`, `num_samples` to match swift naming conventions (camel case and "count" instead of "num").

Reply:
Oh sorry, didn't notice this earlier: The indentation should follow the conventions at https://google.github.io/swift/#parameter-returns-and-throws-tags